docs/source/index.rst

```diff
@@ -181,10 +181,10 @@ conversion utilities for the following models:
 
     task_summary
     model_summary
+    tokenizer_summary
     preprocessing
     training
     model_sharing
-    tokenizer_summary
```

Comment:
I would still leave this here. For me preprocessing should come before the tokenzier summary so people know what a tokenizer is, and IMO, training and model sharing are easier and more import subjects.